syntax = "proto3";

// Client: Server here is my file or my command to delete one
message FilePacket {
  string file_path = 1;
  enum FileCommand {
    CREATE_MODIFY = 0;
    ERASE = 1;
  }
  FileCommand command = 2;
  int64 file_size = 3;
  repeated bytes file_chuncks = 4;
  int64 file_last_write_time = 5;
}

// Client: Server what is your last_time_write for this file?
message ProbeSingleFileRequest {
  string file_path = 1;
}

// Server: Client last_time_write for this file is
message ProbeSingleFileResponse {
  string file_path = 1;
  int64 file_last_write_time = 2;
}